### DISCUSSION TITLE
how to get request protocol?

### DISCUSSION THREAD

**Discussion by KeJunMao** [nuxt/nuxt#15956] [Questions]
in Brower, i can use `location.protocol`. in Express, i can use `req.protocol`

in Nuxt3, i can use `useRequestEvent` to get request, but `event.node.req.protocol` is undefined

**Comment by xjccc** [comment]
nuxt/framework#9646 is the similar？

**Reply by KeJunMao** [reply]
No, `host` only has domain name
